lib/urlwatch/main.py

```diff
@@ -68,6 +68,21 @@ def __init__(self, urlwatch_config, config_storage, cache_storage, urls_storage)
         if hasattr(self.urlwatch_config, 'migrate_urls'):
             self.urlwatch_config.migrate_cache(self)
 
+    def should_run(self, idx, job):
```

Comment:
This is nice, as the expression in the `worker.py` list comprehension was already getting kind of unwieldy, thanks :)